### PR TITLE
Improve logging by introducing slf4j and logback

### PR DESCRIPTION
**Pull request by dennisgranath**
### Description
Change logging from java util logging (jul) to slf4j. Add option to log in jsonFormat with command line argument --logJson. All "native" selenium logs are sent to logback using jul-to-slf4j by setting the SLF4JBridgeHandler (logging_info.properties). In order to that in a smooth way i added the selenium jar into the resulting zalenium.jar.

### Motivation and Context
We would like to be able to log in json output and add support for MDC (mapped diagnotic context). Using this approach one could build dashboards in tools like for example kibana. 

### How Has This Been Tested?
- Unit tests run
- Manual testing of DockerSeleniumStarterProxy and BrowserStackProxy

### Types of changes

- [x] New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by diemol** [comment]
Thanks @dennisgranath for this PR! 
I'll run a few tests and then merge it to a protected branch so Travis can run all the tests. I'll get back to you soon.

File: pom.xml [hunk 1]
```diff
@@ -260,32 +287,7 @@
                                 </filter>
                             </filters>
                             <artifactSet>
```
Comment (diemol) [inline pom.xml]: This makes so much sense! I don't know how we overlooked that :) Thanks!

**Comment by diemol** [comment]
Merging to a protected branch to run CI tests.

File: src/main/resources/logging_info.properties [hunk 2]
```diff
@@ -16,8 +16,7 @@
 # By default we only configure a ConsoleHandler, which will only
 # show messages at the INFO and above levels.
 #handlers= java.util.logging.ConsoleHandler
-handlers= java.util.logging.FileHandler, java.util.logging.ConsoleHandler
-
+handlers = org.slf4j.bridge.SLF4JBridgeHandler
```
Comment (dennisgranath) [inline src/main/resources/logging_info.properties]: Just one note here. If someone is dependent on that logs are written to a file I think we need to add an optional filehandler in logback.xml.

**Pull request merged**